ctapipe/ml/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from ctapipe.utils.datasets import resource_file
+from ctapipe.core import run_tool
+
+
+@pytest.fixture(scope="session")
+def model_tmp_path(tmp_path_factory):
+    return tmp_path_factory.mktemp("models")
+
+
+@pytest.fixture(scope="session")
+def energy_regressor_path(model_tmp_path):
+    from ctapipe.ml.tools.train_energy_regressor import TrainEnergyRegressor
+
+    tool = TrainEnergyRegressor()
+    config = resource_file("ml_config.yaml")
+    out_file = model_tmp_path / "energy.pkl"
```

Comment:
We should use the `FileLock`, as in other tests

Reply:
done